### PR TITLE
fix: filter postRender promises

### PR DESCRIPTION
**Pull request by yohanlaborda**
# Pull request summary

Related to [MS-1869](https://oat-sa.atlassian.net/browse/MS-1869).

# Checklist

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not master

### CODE REVIEW

File: src/runner/provider/qti.js [hunk 1]
```diff
@@ -113,10 +113,16 @@ var qtiItemRuntimeProvider = {
                     }
                 }
 
+                const itemPostRender = this._item.postRender(options);
+                let postRenderPromises = [];
+                if (itemPostRender.length) {
+                    postRenderPromises = itemPostRender.filter(render => render instanceof Promise);
```
Comment (krampstudio) [inline src/runner/provider/qti.js:119]: question: since`Promise.all` supports non-Promise items, why do you filter them out?

**Review by krampstudio — CHANGES_REQUESTED** [review]
Can you please provide some context: 
 - what you are trying to solve and how you try to solve it.
Can it be covered by unit tests? 

Based on the task linked to the ticket this change alone will hide the error. Usually, we use `TypeError` for  implementation issues.  
In the task the error is about a missing container, can you check why the container is missing in your context?

File: src/runner/provider/qti.js (hunk 1 above)
Comment (yohanlaborda) [reply src/runner/provider/qti.js:119]: When the promise tries to call and you send other elements, it calls the error, if you remove the other elements and just pass the promise it does not throw the error.

**Review by btamas — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request closed without merging**